### PR TITLE
Restrict which fields to show in search results

### PR DESCRIPTION
**Pull request by lfarrell**
Restrict which fields to show in search results and truncate long field values
https://jira.lib.unc.edu/browse/HYC-313

### CODE REVIEW

File: app/controllers/catalog_controller.rb [hunk 1]
```diff
@@ -72,25 +72,10 @@ def self.modified_field
...
-    config.add_index_field solr_name("proxy_depositor", :symbol), label: "Depositor", helper_method: :link_to_profile
-    config.add_index_field solr_name("depositor"), label: "Owner", helper_method: :link_to_profile
-    config.add_index_field solr_name("publisher", :stored_searchable), itemprop: 'publisher', link_to_search: solr_name("publisher", :facetable)
-    config.add_index_field solr_name("based_near_label", :stored_searchable), itemprop: 'contentLocation', link_to_search: solr_name("based_near_label", :facetable)
-    config.add_index_field solr_name("language", :stored_searchable), helper_method: :language_links, link_to_search: solr_name("language", :facetable)
-    config.add_index_field solr_name("date_uploaded", :stored_sortable, type: :date), itemprop: 'datePublished', helper_method: :human_readable_date
-    config.add_index_field solr_name("date_modified", :stored_sortable, type: :date), itemprop: 'dateModified', helper_method: :human_readable_date
     config.add_index_field solr_name("date_created", :stored_searchable), itemprop: 'dateCreated'
```
Comment (smithjp) [inline app/controllers/catalog_controller.rb]: Do we want date_created or date_issued here?

**Comment by smithjp** [comment]
Something strange is happening with truncated creator names.  I think the creator name is being linked to a facet search which does not work well with truncating.  The link name will need to be truncated instead of the whole html element.
screen shot 2018-10-11 at 12 45 37 pm

File: app/controllers/catalog_controller.rb (hunk 1 above)
Comment (lfarrell) [reply app/controllers/catalog_controller.rb]: Seems like they're not used consistently, but it was what was in the ticket. I'll check with Rebecca.

Comment (lfarrell) [reply app/controllers/catalog_controller.rb]: Fun, hand't thought about it truncating the URL

Comment (lfarrell) [reply app/controllers/catalog_controller.rb]: Should be fixed now. Doesn't make much sense to make it a link if you can't search all the authors.

**Comment by lfarrell** [comment]
Bad force push that missed some changes. Just easier to open a new request

**Pull request closed without merging**